lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -280,8 +294,11 @@ def create_stack(module, stack_params, cfn):
     if 'TemplateBody' not in stack_params and 'TemplateURL' not in stack_params:
         module.fail_json(msg="Either 'template' or 'template_url' is required when the stack does not exist.")
 
-    # 'disablerollback' only applies on creation, not update.
+    # 'disablerollback' and 'EnableTerminationProtection' only
+    # apply on creation, not update.
     stack_params['DisableRollback'] = module.params['disable_rollback']
+    stack_params['EnableTerminationProtection'] = bool(module.params.get(
+        'termination_protection'))
```

Comment:
this line break is .. awful. Can you find another pep8 way to do it? maybe start the second line with 'module'

Reply:
looks like lines can be as long as 160 chars. So removed some unnecessary line-breaks.